charts/castai-cluster-controller/values.yaml

```diff
@@ -49,3 +49,7 @@ serviceAccount:
 
 aks:
   enabled: false
+
+priorityClass:
+  enabled: true
```

Comment:
will you set disabled for older versions?

Reply:
We will need to disable it only for old clusters during cluster-controller upgrade and customers who are using old k8s versions.